user/server.js

```diff
@@ -331,6 +361,88 @@ function mongoLoop() {
 
 mongoLoop();
 
+function getData() {
+    return new Promise((resolve, reject) => {
+        const size = 1024 * 1024;
+        fs.open('/dev/urandom', 'r', (err, fd) => {
+            if (err) {
+                reject(err);
+            }
+            let buffer = Buffer.alloc(size);
+            fs.read(fd, buffer, 0, size, 0, (err, num) => {
+                if (err) {
+                    reject(err);
+                } else {
+                    fs.close(fd);
+                    resolve(buffer);
+                }
+            });
+        });
+    });    
+}
+
+function randRange(min, max) {
+    return (Math.random() * (max - min)) + min;
+}
+
+var hog = [];
+var hashCount = 0;
+function memoryHog(size) {
+    const hogMaxLength = 40;
```

Comment:
Consider making this configurable through an environment property

Reply:
I did a lot of testing and tuning and this is the best value.